.travis.yml

```diff
@@ -1,37 +1,18 @@
 language: ruby
+cache: bundler
 rvm:
-  - 2.1.10
-  - 2.2.1
-  - 2.2.2
-  - 2.2.10
-  - 2.3.8
-  - 2.4.9
   - 2.5.7
   - 2.6.5
   - 2.7.0
```

Comment:
What do you think about testing the latest minor version instead of explicit version?

Reply:
Good idea. I will do that

Reply:
I got a wired result:

* [2.5 points to 2.5.5](https://travis-ci.org/github/jeremytregunna/ruby-trello/jobs/681528450)
* [2.6 points to 2.6.3](https://travis-ci.org/github/jeremytregunna/ruby-trello/jobs/681528453)

Maybe I should change back. Or did I do something wrong?

Reply:
@wokenqingtian let's report that to Travis CI team instead

Reply:
OK, sure

Reply:
After report this issue, Travis team [fixed it](https://github.com/travis-ci/travis-build/pull/1902) immediately. I also checked our recent CI logs. All point to correct Ruby versions.